src/test/java/com/crackingthecodeinterview/chapter1/CheckPermutationTest.java

```diff
@@ -0,0 +1,34 @@
+package com.crackingthecodeinterview.chapter1;
+
+import org.junit.jupiter.api.Test;
+
+import static com.crackingthecodeinterview.utilities.Constants.EMPTY_STRING;
+import static com.crackingthecodeinterview.chapter1.CheckPermutation.checkPermutation;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CheckPermutationTest {
+    @Test
+    void checkPermutationTest_OnNoNullStrings_ture() {
```

Comment:
No make sense! Think about scenarios not just copy! checkPermutationTest_OnPermutedStrings_ture